components/BannerNotificationContainer.tsx

```diff
@@ -29,6 +34,18 @@ export const BannerNotificationContainer: React.FC = () => {
         </View>
       )}
 
+      {showQuickShareSuccessBanner && (
+          <View style={{marginTop: 10, marginBottom: 10}}>
+            <BannerNotification
+                type={BANNER_TYPE_SUCCESS}
+                message={rt('status.accepted.message')}
+                onClosePress={scanScreenController.DISMISS_QUICK_SHARE_BANNER}
+                key={'activatedVcPopup'}
+                testId={'activatedVcPopup'}
+            />
```

Comment:
Why key and testId are activatedVcPopup?

Reply:
Updated plz take a look